### ISSUE TITLE
HTML breadcrumbs issue

### ISSUE THREAD

**Issue by mskelton** [one-dark/jetbrains-one-dark-theme#105]
Copied from plugin review page:

> I found a bug in your thread. The bottom line is the incorrect selection of the HTML tag tree. In the color profile file, these are options for breadcrumbs. I copied the missing parameters from the Atom One Dark file and it worked.
>
> These options are:

```xml
 <value>
    <option name="FOREGROUND" value="ffffff" />
    <option name="BACKGROUND" value="59626f" />
  </value>
</option>
<option name="BREADCRUMBS_DEFAULT">
  <value>
    <option name="FOREGROUND" value="c6c6c6" />
    <option name="BACKGROUND" value="282c34" />
  </value>
</option>
<option name="BREADCRUMBS_HOVERED">
  <value>
    <option name="FOREGROUND" value="ffffff" />
    <option name="BACKGROUND" value="2979ff" />
  </value>
</option>
<option name="BREADCRUMBS_INACTIVE">
  <value>
    <option name="FOREGROUND" value="6b727d" />
    <option name="BACKGROUND" value="282c34" />
  </value>
</option>
```

![inOVWZV](https://user-images.githubusercontent.com/25914066/68966382-8e7f6a00-07a3-11ea-8219-8529ed7b5ef3.png)

**Comment by mindyourlifeguide** [comment]
I have been using your theme from the very beginning. Rather, code highlighting. As for me, she is much better than in Atom One Dark Theme. Although many things I have already customized for myself regarding the highlight. As a theme, I installed Material Theme UI and set my interface colors to more contrast the menu and the workspace. Now to the bug that I discovered. 
I’m tracking one developer on [youtube](https://www.youtube.com/watch?v=20UsCQzUOIE&t=100s) and saw him have a more convenient highlighting of breadcrumbs. At first glance, he used the same theme as mine. I asked him which extension he was using, to which he replied that this was just a theme.I started looking for the problem, reviewed all the settings, but everything looked gray. 
After I installed Atom One Dark Theme and the highlight worked there, although the background was highlighted, not the text.I went to theme settings in explorer and found settings that I didn't have. I wrote a review about this.
But I decided to go deeper, deleted the entire config and installed your theme cleanly. Highlight of breadcrumbs worked, but now the letters were already highlighted, as in the video, and not the background. And now I think that most likely it was my highlight file that was broken.

Right after the configuration based on Atom One Dark Theme:
![3333](https://user-images.githubusercontent.com/35348648/68973718-a2ed5200-07f7-11ea-994b-e521c94efb09.png)

Here, after the configuration based on your theme. Only the colors made a little brighter.

![3444](https://user-images.githubusercontent.com/35348648/68973709-9c5eda80-07f7-11ea-9079-f0b00eea266b.png)

Thank you for your work!

**Comment by mskelton** [comment]
@mindyourlifeguide Thanks for the detailed write up!  I'm not sure I'm entirely following what the issue is especially since this theme does not add any colors to the breadcrumbs (see screenshot).  Is it possible this is another plugin that is adding the breadcrumb colors?  Unless it is in an IDE that I'm not using (i.e. WebStorm)?

One thing that appears different in your screenshot is the HTML bracket colors which could be why there is the difference in the breadcrumb colors.

![2019-11-16_04-40-23](https://user-images.githubusercontent.com/25914066/68992038-879c3a00-082b-11ea-8b3e-02f9625418ef.png)

**Comment by mindyourlifeguide** [comment]
I use webstorm. The problem was that my breadcrumbs were not highlighted, although the highlight item was turned on in the settings.

This highlight really gives your color theme. For myself, I made the colors a little brighter.

![999-2](https://user-images.githubusercontent.com/35348648/68992318-6448ad80-0872-11ea-8781-61ea1f76a61b.png)
![999-1](https://user-images.githubusercontent.com/35348648/68992337-a5d95880-0872-11ea-9fe7-34aede4f6f33.png)




To really change the color of the brackets, I really use the [Rainbow Brackets](https://plugins.jetbrains.com/plugin/10080-rainbow-brackets/) plugin, and to highlight them correctly, as I had before in VS Code, I use [HighlightBracketPair](https://plugins.jetbrains.com/plugin/10465-highlightbracketpair/).

The essence of the problem was in the broken color settings file. How this happened I don`t know. I just  filmed all the settings items that I ruled for myself on the video, deleted everything, installed it again and configured it using the video. Now everything works.

The bug is not in your color theme, but in my broken settings file.

I will delete that incorrect review and write a new one.

I'm sorry to trouble you!
